ui/txs/noves/useDescribeTxs.tsx

```diff
@@ -0,0 +1,89 @@
+import type { UseQueryResult } from '@tanstack/react-query';
+import _ from 'lodash';
+
+import type { NovesDescribeTxsResponse } from 'types/api/noves';
+import type { Transaction } from 'types/api/transaction';
+
+import type { ResourceError } from 'lib/api/resources';
+
+export interface DescribeTxs {
+  txHash: string;
+  type?: string;
+  description?: string;
+  isLoading: boolean;
+  enabled: boolean;
+}
+
+export interface TransactionWithTranslate extends Transaction {
+  translate?: DescribeTxs;
+}
+
+export default function useDescribeTxs(items: Array<Transaction> | undefined, translateEnabled: boolean) {
+  const txsHash = items?.map(i => i.hash);
+  const txsChunk = _.chunk(txsHash || [], 10);
```

Comment:
The uniqueness of each item should also be checked, as there may be multiple rows with the same transaction hash on a single page.